### PR TITLE
Hasan/Translation 1426 prime commercial plan

### PR DESCRIPTION
**Pull request by hasan-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/features/pages/partners/deriv-prime/commercial-plan/index.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { commercial_table } from './styles.module.scss'
+import Container from 'features/components/atoms/container'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+import Table from 'features/components/atoms/table'
+import dclsx from 'features/utils/dclsx'
+
+const tableData = [
```
Comment (aswathy-deriv) [inline src/features/pages/partners/deriv-prime/commercial-plan/index.tsx:9]: Can we change the data name to `snake_case` eg: table_data

Comment (habib-deriv) [inline src/features/pages/partners/deriv-prime/commercial-plan/index.tsx:9]: Let's have a separate data file for the data and import it here

File: src/features/styles/theme/table.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
+@import 'features/styles/theme/variables';
+
+.table-responsive {
+    display: block;
+    width: 100%;
```
Comment (mayuran-deriv) [inline src/features/styles/theme/table.scss:5]:
```suggestion
    inline-size: 100%;
```

File: src/features/pages/partners/deriv-prime/commercial-plan/index.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    {
+        '_t_Monthly volume requirement_t_': <Localize translate_text="_t_0 - 3 billion_t_" />,
+        '_t_Fee per million_t_': <Localize translate_text="_t_$ 8.00_t_" />,
+    },
+    {
+        '_t_Monthly volume requirement_t_': <Localize translate_text="_t_> 3 - 5 billion_t_" />,
```
Comment (aswathy-deriv) [inline src/features/pages/partners/deriv-prime/commercial-plan/index.tsx:15]: Can you format the data structure like `column_data` with all the column values, and `row_data`with all the row values than repeating the Monthly volume requirement and fee per million many times

File: src/features/components/atoms/table/base/index.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+const BaseTable = ({ data, striped, className, ...rest }: TableProps) => {
+    return (
+        <Box
+            className={dclsx('table-responsive', { ['table-striped']: striped }, className)}
+            {...rest}
+        >
+            <table className={dclsx('table')}>
+                <thead>
+                    <tr>
+                        {Object.keys(data[0]).map((key) => (
```
Comment (mayuran-deriv) [inline src/features/components/atoms/table/base/index.tsx:24]: better to split the data structure, as column data, and row data  so we can avoid these kinds of complex mappings 
just a suggestion

File: src/features/styles/theme/table.scss (hunk 2 above)
Comment (hasan-deriv) [reply src/features/styles/theme/table.scss:5]: Done

File: src/features/styles/theme/table.scss [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    overflow-x: auto;
+    -webkit-overflow-scrolling: touch;
+}
+.table {
+    width: 100%;
```
[the remark this answers is not in the document]
Comment (hasan-deriv) [reply src/features/styles/theme/table.scss:10]: done

File: src/features/pages/partners/deriv-prime/commercial-plan/index.tsx (hunk 1 above)
Comment (hasan-deriv) [reply src/features/pages/partners/deriv-prime/commercial-plan/index.tsx]: Sure..

File: src/features/pages/partners/deriv-prime/commercial-plan/index.tsx (hunk 3 above)
Comment (hasan-deriv) [reply src/features/pages/partners/deriv-prime/commercial-plan/index.tsx]: Done...

File: src/features/components/atoms/table/base/index.tsx (hunk 4 above)
Comment (hasan-deriv) [reply src/features/components/atoms/table/base/index.tsx]: Done..

File: src/features/pages/partners/deriv-prime/commercial-plan/index.tsx (hunk 1 above)
Comment (hasan-deriv) [reply src/features/pages/partners/deriv-prime/commercial-plan/index.tsx]: Done..

**Review by mayuran-deriv — APPROVED** [review]

**Review by aswathy-deriv — APPROVED** [review]

**Pull request merged**